geopolars/src/geoseries.rs

```diff
@@ -70,6 +70,9 @@ pub trait GeoSeries {
     /// Applies to GeoSeries containing only Polygons. Returns `None` for other geometry types.
     fn exterior(&self) -> Result<Series>;
 
+    /// Explodes multi-part geometries into multiple single geometries.
+    fn explode_geometries(&self) -> Result<Series>;
```

Comment:
I see that polars [already has an `explode` function](https://pola-rs.github.io/polars/py-polars/html/reference/api/polars.Series.explode.html?highlight=explode#polars.Series.explode), but in this case, the meaning is the same for both, so maybe we could call this fn `explode` as well?

Reply:
Addressed in https://github.com/geopolars/geopolars/pull/111/commits/28dd535bba9cf9d0126b21268240a60426a5143d